express/blocks/template-list/template-list.js

```diff
@@ -27,19 +27,22 @@ import { buildCarousel } from '../shared/carousel.js';
 
 const cache = {
   templates: [],
-  queryString: '',
+  total: 0,
+  type: 'flyer',
```

Comment:
I would not define a default here... this field is mandatory and if not set, the block should probably output an error message.